DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py

```diff
@@ -375,7 +375,7 @@
     process.multFilter = HLTrigger.special.hltPixelActivityFilter_cfi.hltPixelActivityFilter.clone(
         inputTag  = 'siPixelClustersPreSplitting',
         minClusters = 150,
-        maxClusters = 50000 # was 10000
+        maxClusters = 70000 # was 10000
```

Comment:
```suggestion
        maxClusters = 70000 # was 50000
```

Reply:
Changed in https://github.com/cms-sw/cmssw/pull/43036/commits/5475a0fb60e27d29bf5b189298d92e95458ec677